themis/spec/common/soter-container_test.go

```diff
@@ -0,0 +1,31 @@
+package common
+
+import (
+	"encoding/base64"
+	"testing"
+)
+
+func TestExampleContainer(t *testing.T) {
```

Comment:
imho, better to test container implementation using C themis library. Generate some container by source of trust library and try to parse it here to proof that it is correct implementation and it is compatible with C themis. Because without this check it is one more container that lives separately from everyone.

Reply:
> imho, better to test container implementation using C themis library

That's exactly what is being done here: the container is a public key that I have generated with actual Themis.

I believe that a proper interoperability test suite can be implemented later, but it should stay in a separate package from this one. That will allow to keep *this* implementation with exactly zero external dependencies, immediately available to be `go run` or `go test`. A test suite for interoperability with actual GoThemis will require Themis Core to be installed in the system, at least. I don't want this to be a prerequisite to step through a test you're interested in when reading the spec.

However, this is really nice idea to implement later: ensure that both the reference and actual implementation behave the same, at least on the happy path. Thanks for pointing it out!

Reply:
oh, I didn't know that base64 value is container generated by C themis. can you add comment about it? or describe in comment how to generate the same. For example using gothemis in golang and what to copy\paste for that.

Reply:
> can you add comment about it?

Well, sure, here you go.

I did not generate *this* particular value in the described way, but command-line tools can be most definitely used to generate a new key and wrap it in base64 for compactness.